src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -28,14 +24,21 @@
 
 			// Test if sum is equal
```

Comment:
think about extracting methods for all inline comments -> then the code is more self explaining